mirrord/config/src/feature.rs

```diff
@@ -76,12 +78,20 @@ pub struct FeatureConfig {
     /// ## feature.network {#feature-network}
     #[config(nested, toggleable)]
     pub network: NetworkConfig,
+
+    /// ## feature.copy_target {#feature-copy-target}
+    ///
+    /// Creates a new copy of the target. This copy will be used instead of the original target.
+    /// This is feature requires mirrord operator.
+    #[config(env = "MIRRORD_COPY_TARGET_ENABLED", default = false, unstable)]
```

Comment:
I think at some point @aviramha wanted us to stop adding new env vars where not necessary, right?
```suggestion
    #[config(default = false, unstable)]
```

Reply:
That's true. Removed the env var, we can always add it later